package.json

```diff
@@ -8,6 +8,7 @@
     "build": "./node_modules/.bin/gatsby build --prefix-paths",
     "develop": "./node_modules/.bin/gatsby develop",
     "format": "./node_modules/.bin/prettier --write src/**/*.{js,jsx} tests/*.js gatsby-node.js",
+    "format:uc-doc": "./node_modules/.bin/prettier --write content/uc-doc/*.md content/uc-doc/**/*.md",
```

Comment:
Not sure if that said that we cannot add file with more than 2 layers (without updating this file)? If so, it's pretty dangerous as configuration

Reply:
Fixed an issue with the script... `globstar` was not working well when running with `yarn`. Yes, this script is checking on multiple level for markdown files.

> a globstar ** should match "zero or more directories and subdirectories searching for matches".

Can you explain why it's pretty dangerous?